### DISCUSSION TITLE
RingDAO Marketing Action Plan 2024-2025

### DISCUSSION THREAD

**Discussion by nblogist** [ringdao-community-guild/discussion#169] [General]
Hello all!

I have completed the first version of the marketing action plan, I have given much thought to it, ensuring practicality and optimizing for faster results and easier implementation, obviously with anything human, this requries inprovements and feedback.

Please find the document [here](https://docs.google.com/document/d/1DfFt0wVNb7yTALMX_u7U0AtWx1r-fVp2nUrDWgPKkMM/edit), and feel free to comment. respond to existing comments and if you want to start a discussion on a certain point just start one here in the comments.

LFG! :rocket: 

Furqan A.

CC @hackfisher

**Comment by nblogist** [comment]
@PartnershipDude can you please tag everyone in marketing?

**Reply by nblogist** [reply]
Okay! Just wanna make sure everyone gets this!

**Comment by SasoLithops** [comment]
First of all THANK YOU @nblogist for putting this together. Great job!!

So after going through document here are my comments:

Since I have no marketing knowledge background I will stay away for marketing comments but stick to everyday user comments:-)
Before I go point by point I love KPIs. They will trully be direct numbers easy for everyone to check for transparent show of progress. 
Everything I don't comment I agree or have no knowledge to comment.

Point 1.

1.C.b I have just added in our marketing TG group visit to Cosmoverse. Hope this will raise the numbers of active developers. And one major blockchain conference ticked (of course if you guys agree we join)

1. D. Definitely, but this time DCDAO should decide if projects get grant and grant should be paid in few installments each next after proof of progress. But as you said we can discuss this later. This is where treasury loses a lot of funds by funding scam projects,

Point 3.

3.A.a. This will be hard. Getting new utility to Ring has proven impossible in last few years. It only lost utility with end of Evolution Land, I am looking forward to ideas on that level. 

3.A.b. I think we already have very rewarding staking program with last post from Laki showing Ring APR 13-14% and KTON 52%. This is so attractive it actually becomes unatractive as people wonder about sustainibility. I personally would if I had free money now never invest is sush high staking APR token. Checking price movement in Coingecko would only confirm my decision.

Have to add a MUST is 3.C.b KTON is becoming unknown token. No utility beside staking and farming in snowswap what so ever. Main cause for price tank.  Also same as answered in 3.A.b.

3.C.c. is great idea. Changes are welcome after some time, to push to next level. Also would be good to check real number of participants as some have strangley same avatar or from same collection. Laki may provide more details on this. 

4.B. and 5. YES,YES 1000%

Some I commented to your comments in document.

**Reply by nblogist** [reply]
Thank you for your feedback @SasoLithops!
Point 3.A.a
>Introduce new use cases for RING within the Darwinia ecosystem
>3.A.a. This will be hard. Getting new utility to Ring has proven impossible in last few years. It only lost utility with end of Evolution Land, I am looking forward to ideas on that level.

Yes, true but we should think outside of the box then, box being blockchain. could be RWA - or simple buy xyz using Ring (again, if we can't have something in box, perhaps we can come up with something outside, and it doesn't have to be efficient, it just has to be, be it advantages for certain apps that developers use, if they were to use RIng to pay for etc.)


>![image](https://github.com/user-attachments/assets/5e107c18-c3a9-4881-b987-dbf763fceefc)
>3.A.b. I think we already have very rewarding staking program with last post from Laki showing Ring APR 13-14% and KTON 52%. This is so attractive it actually becomes unatractive as people wonder about sustainibility. I personally would if I had free money now never invest is sush high staking APR token. Checking price movement in Coingecko would only confirm my decision.

Thank you for pointing it out @SasoLithops I have added another point now to make sure rates go down.

>Have to add a MUST is 3.C.b KTON is becoming unknown token. No utility beside staking and farming in snowswap what so ever. Main cause for price tank. Also same as answered in 3.A.b.
@SasoLithops do you have suggestions to improve on this?


Thank you for taking the time to respond @SasoLithops ! you help a lot!

**Comment by nblogist** [comment]
This week I will move it for a vote! Sharing on TG again!

**Comment by SasoLithops** [comment]
_Have to add a MUST is 3.C.b KTON is becoming unknown token. No utility beside staking and farming in snowswap what so ever. Main cause for price tank. Also same as answered in 3.A.b.
@SasoLithops do you have suggestions to improve on this?_

I will propose this and some ideas on tokenomics for discussion in Github later this month.

**Reply by nblogist** [reply]
once we get this voted on we can later then - when its time to tackle the particular problem can discuss methods to improve- or decide on a good execution.

**Comment by JuneCribb** [comment]
This is great and with achievable KPI. 

I love the part where we bring more people to build on Darwinia with the grant program. 

Very incentivizing.

**Reply by nblogist** [reply]
Thank you June!! It will be amazing if you can vote on tally!!

**Comment by SasoLithops** [comment]
Just posting here so we have in on place regarding 3.C.b discussion:

https://github.com/orgs/darwinia-network/discussions/1566

**Comment by Divblaze** [comment]
I think a hackathon should be done on Darwinia network to bring in developers and also get some external products on Darwinia network

**Reply by nblogist** [reply]
Agreed! We have it in pipeline to eventually get to it - currently we are making the base we need to handle influx of new devs, like the [Developer Video Series](https://github.com/orgs/dcdao/discussions/167#discussioncomment-10184859) here.

**Reply by Divblaze** [reply]
That awesome, would check the Videos out

**Reply by nblogist** [reply]
Thank you for your support!
